spec.bs

```diff
@@ -221,6 +226,55 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
     [=interest group ad/render url=].
   1. The [=string/length=] of |ad|'s [=interest group ad/metadata=] if the field is not null.
 
+<div algorithm>
+
+To <dfn>check interest group permissions</dfn> given an [=url/origin=]

```

Comment:
I think these should be just `[=origin=]` and same elsewhere in this PR _unless_ you're actually taking something of type URL and getting _its_ specific origin, whose type is `[=origin=]`